plugins/sharding/logical_range_filter.rb

```diff
@@ -40,7 +40,7 @@ def run_body(input)
               n_elements += result_set.size
 
               if is_first
-                writer.open_result_set(result_set, output_columns, -1)
+                writer.open_result_set_metadata(result_set, output_columns, -1)
```

Comment:
`records:`の分で`1`じゃない？

```suggestion
                writer.open_result_set_metadata(result_set, output_columns, 1)
```

Reply:
タシカニ、と思って `1` にしたら `n_hits` の値でした。
ので、 `..., -1, 1)` で `n_additional_elements` を指定しました。